package.json

```diff
@@ -10,7 +10,9 @@
       "import": "./index.mjs"
     },
     "./aws": "./lib/aws/index.js",
-    "./": "./"
+    "./*": "./",
+    "./opensearch/*": "./lib/opensearch/*",
+    "./lib/*": "./lib/*"
```

Comment:
Are these changes related to this fix? Rebase your branch.

Reply:
No, they aren't. It was intended to fix deprecation mapping error when using NodeJS 16 and TypeScript https://github.com/opensearch-project/opensearch-js/issues/365. I am removing this fix for now, leaving it as it was previously.

Reply:
> Are these changes related to this fix? Rebase your branch.

No, they aren't. It was intended to fix deprecation mapping error when using NodeJS 16 and TypeScript https://github.com/opensearch-project/opensearch-js/issues/365. I am removing this fix for now, leaving it as it was previously.